"What is Ethereum" page copy changes (#888)
ethereum/ethereum-org-website: docs/what-is-ethereum/index.md

**Launched in 2015, Ethereum is the world’s programmable blockchain.**

**Like other blockchains, Ethereum has a native cryptocurrency called Ether (ETH).** [ETH is digital money](/eth/). If you’ve heard of [Bitcoin](http://bitcoin.org/), ETH has many of the same features. It is purely digital, and can be sent to anyone anywhere in the world instantly. The supply of ETH isn’t controlled by any government or company - it is decentralized, and it is scarce. People all over the world use ETH to make payments, as a store of value, or as collateral.

**But unlike other blockchains, Ethereum can do much more.** Ethereum is programmable, which means that developers can use it to [build new kinds of applications](/dapps/).

These decentralized applications (or “dapps”) gain the benefits of cryptocurrency and blockchain technology. They are reliable and predictable, meaning that once they are “uploaded” to Ethereum, they will always run as programmed. They can control digital assets in order to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them.

> **Right now, thousands of developers all over the world are building applications on Ethereum, and inventing new kinds of applications. Today, people aroun**

Reviewer: `Today, people aroun` ?

Author: ah, my bad. I would just revert to:

```suggestion
**Right now, thousands of developers all over the world are building applications on Ethereum, and inventing new kinds of applications, many of which you can us today:**
```
